dqc_us_rules/resources/DQC_US_0001/dqc_0001.json

```diff
@@ -339,7 +353,13 @@
 			"SICAD1Member", 
 			"SICAD2Member", 
 			"OtherCurrencyMember", 
-			"NonUSDollarMember"
+			"NonUSDollarMember",
+		    "CENCOEXMember",
```

Comment:
I fixed this with commit: 10444ac1322da0cd0f026e6dfad83aa13ab56e97
It's included in this PR but comes after this commit.